beastify/popup/choose_beast.js

```diff
@@ -77,5 +76,22 @@ browser.tabs.executeScript({file: "/content_scripts/beastify.js"})
         .catch(reportError);
     }
   });
-  
-});
+}
+
+/**
+ * There was an error executing the script.
+ * Display the popup's error message, and hide the normal UI.
+ */
+function reportExecuteScriptError() {
```

Comment:
Not: Maybe add an error parameter and then

    console.error(`Failed to beastify: ${errror.message}`);

so that beginners can know how to see error messages for debugging?